docs/userGuide/tweakingThePageStructure.md

```diff
@@ -148,20 +147,46 @@ If you wish insert scripts at the bottom, before MarkBind's scripts, simply inse
 
 ---
 
-### Fixing the header to the top
+### Sticking the header to the top
 
-Headers are commonly included inside the HTML `<header>` tag. In encouraging this, a convenient interface to implement <tooltip content="Headers that stick to the top of the page while scrolling the content">fixed headers</tooltip> surrounding the `<header>` tag is provided that ensures page anchors work correctly.
+Headers are commonly included inside the html `<header>` tag. In encouraging this, 2 convenient options to implement fixed/sticky headers using the `<header>` tag are provided, which primarily ensures that the page **scrolls to anchors** correctly.
```

Comment:
When reading this, I am confused as to why we explain that the convenient options "encourage" the use of headers within the `<header>` tag 😓, since they are provided **primarily** to ensure 
> that the page **scrolls to anchors** correctly.